### PR TITLE
Improve organ and case counts displayed on the homepage

### LINKED ISSUE #681: Homepage Fixes
- [x] Make this 21 (coarse organ grouping rather than detailed organ): 
image
- [x] change " includes tumors originating from 21 primary tumor sites" to " includes tumors originating from 21 tumor sites"
- [x] Minor other bug: Case count is different on the homepage vs explore page (2,040 vs 2,088)

### PR DESCRIPTION
**Pull request by onursumer**
Fix #681

image

TODO: Biopsecimen count still does not match

### CODE REVIEW

File: packages/data-portal-commons/src/lib/entityReportHelpers.ts [hunk 1]
```diff
@@ -49,12 +51,14 @@ export function normalizeTissueOrOrganOrSite(value: string) {
...
-      return trimmedTreatment.charAt(0).toUpperCase() + trimmedTreatment.slice(1);
+    const treatments = value.split(/,|;/).map((t) => {
+        const trimmedTreatment = t.trim();
+        return (
+            trimmedTreatment.charAt(0).toUpperCase() + trimmedTreatment.slice(1)
+        );
     });
     return _.uniq(treatments);
-  }
+}
```
Comment (onursumer) [inline packages/data-portal-commons/src/lib/entityReportHelpers.ts:61]: This is a `prettier` change unrelated to this PR.

File: packages/data-portal-commons/src/lib/entityReportHelpers.ts [hunk 2]
```diff
@@ -134,11 +138,9 @@ export function getNormalizedOrgan(entity: Entity) {
...
 
 export function getNormalizedTreatment(entity: Entity) {
-    return entity.TreatmentType
-      ? normalizeTreatment(entity.TreatmentType)
-      : [];
-  }
-  
+    return entity.TreatmentType ? normalizeTreatment(entity.TreatmentType) : [];
+}
+
```
Comment (onursumer) [inline packages/data-portal-commons/src/lib/entityReportHelpers.ts:143]: Another `prettier` change unrelated to this PR.

File: lib/helpers.ts [hunk 3]
```diff
@@ -143,18 +145,26 @@ export function computeDashboardData(files: Entity[]): EntityReport[] {
...
+        for (const biospecimen of file.biospecimen) {
+            uniqueBiospecs.add(biospecimen.BiospecimenID);
+        }
+        for (const diagnosis of file.diagnosis) {
+            uniqueCases.add(diagnosis.ParticipantID);
+            uniqueOrgans.add(getNormalizedOrgan(diagnosis));
         }
-        for (const diag of file.diagnosis) {
-            uniqueCases.add(diag.ParticipantID);
-            uniqueOrgans.add(diag.TissueorOrganofOrigin);
```
Comment (inodb) [inline lib/helpers.ts:155]: @onursumer so if i understand the `diag.TissueorOrganofOrigin` field is incorrect? What's the diff between using `getNormalizedOrgan`?

**Review by inodb — APPROVED** [review]
Thanks for fixing!

**Pull request merged**